lib/yardstick/rule_description/classifier.rb

```diff
@@ -0,0 +1,105 @@
+module Yardstick
+  # Rule description composed of tokens which can be formatted
+  class RuleDescription
+    # Token classification
+    module Classifier
+      # List of classifiers
+      class List
+        extend Forwardable
+        include Concord.new(:classifiers), Enumerable
+
+        # Classify a token by returning the first match
+        #
+        # @param token [String] markup text
+        #
+        # @return [Token]
```

Comment:
Ahh ok, yeah, if it's impossible for no classifiers to match then I wouldn't make things more complex.

You could probably use `break` to return the classified value though, eg:

```ruby
def classify(token)
  classifiers.each do |classifier|
    classified = classifier.try(token)
    break classified if classified
  end
end
```